src/core/lib/iomgr/tcp_client_posix.cc

```diff
@@ -138,6 +168,7 @@ static void on_writable(void* acp, grpc_error_handle error) {
   grpc_closure* closure = ac->closure;
   std::string addr_str = ac->addr_str;
   grpc_fd* fd;
+  bool connect_cancelled = false;
```

Comment:
Might as well just declare this on line 184, where it actually gets used.

Reply:
Fixed.